<ui version="4.0" >
 <class>AspellPluginBase</class>
 <widget class="QDialog" name="AspellPluginBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="fmainTab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="ftab1" >
      <attribute name="title" >
       <string>Spell-check</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="ftextLabel1" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>75</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>75</width>
               <height>35</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text" >
              <string>Mis-spelling:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="fmisSpelling" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>300</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>35</height>
              </size>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="ftextLabel2" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>82</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>82</width>
               <height>35</height>
              </size>
             </property>
             <property name="text" >
              <string>Replacement:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="fcurrWord" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>295</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>295</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="flistReplacements" />
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="ftextLabel4" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>123</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>123</width>
               <height>29</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>Active dictionary: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ftextDictDetails" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="ftextLabel3" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>139</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>139</width>
               <height>35</height>
              </size>
             </property>
             <property name="text" >
              <string>Personal
Dictionary</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="faddWordBtn" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="text" >
              <string>&amp;Add Word</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <item>
            <widget class="QPushButton" name="fchangeBtn" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="text" >
              <string>&amp;Change</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fchangeAllBtn" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="text" >
              <string>C&amp;hange All</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QPushButton" name="fskipBtn" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="text" >
              <string>&amp;Skip</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fskipAllBtn" >
             <property name="minimumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>139</width>
               <height>52</height>
              </size>
             </property>
             <property name="text" >
              <string>S&amp;kip All</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ftab2" >
      <attribute name="title" >
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="ftextLabel6" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Available aspell dictionaries:</string>
         </property>
         <property name="textFormat" >
          <enum>Qt::LogText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="flistDicts" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="fcloseBtn" >
       <property name="text" >
        <string>&amp;Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
